src/main/java/school/hei/patrimoine/visualisation/swing/ihm/google/NamedLink.java

```diff
@@ -0,0 +1,3 @@
+package school.hei.patrimoine.visualisation.swing.ihm.google;
+
+public record NamedLink(String name, String value) {}
```

Comment:
String value --> Url link

NamedLink = Name + Link... so make it explicit!

Reply:
done